app/controllers/api/base_controller.rb

```diff
@@ -39,8 +39,8 @@ class BaseController < ActionController::Base
 
     rescue_from Foreman::MaintenanceException, :with => :service_unavailable
 
-    def get_resource
-      instance_variable_get(:"@#{resource_name}") || raise('no resource loaded')
+    def get_resource(options = {})
```

Comment:
@ares, According to [this](https://robots.thoughtbot.com/ruby-2-keyword-arguments) keyword arguments are there since 2.0, so it should be OK for Trusty too.

Unless you are looking for an excuse to drop Trusty support ;)